### PR TITLE
Assignment 1

### PR DESCRIPTION
**Pull request by gloriaconcepto**
hey i know is a little late,so sorry promise to submit in time.

### CODE REVIEW

File: 30daysofscala/src/test/scala/example/HelloSpec.scala [hunk 1]
```diff
@@ -0,0 +1,10 @@
+package example
+
+import org.scalatest.flatspec.AnyFlatSpec
+import org.scalatest.matchers.should.Matchers
+
+class HelloSpec extends AnyFlatSpec with Matchers {
+  "The Hello object" should "say hello" in {
```
Comment (ayojava) [inline 30daysofscala/src/test/scala/example/HelloSpec.scala]: Update the test case to

Comment (ayojava) [inline 30daysofscala/src/test/scala/example/HelloSpec.scala]: Update the test case to also check that hello.greeting returns "Welcome to 30 days of Scala"

**Review by ayojava — COMMENTED** [review]
Made some comments

File: 30daysofscala/src/test/scala/example/HelloSpec.scala (hunk 1 above)
Comment (gloriaconcepto) [reply 30daysofscala/src/test/scala/example/HelloSpec.scala]: ok

**Pull request closed without merging**